Changes

```diff
@@ -132,6 +132,9 @@ Working version
 - GPR#1187: Minimal documentation for compiler plugins
   (Florian Angeletti)
 
+- GPR#1220: Fix "-keep-docs" option in ocamlopt manpage
```

Comment:
I know, credit where credit is due, but do we really need a Changes entry for this? @gasche